### PR TITLE
Clean build image

### PR DESCRIPTION
**Pull request by oryband**

### CODE REVIEW

**Comment by oryband** [comment]
@rounds/server lemme know if you need any of the crap i removed

File: build/Dockerfile [hunk 1]
```diff
@@ -1,32 +1,20 @@
...
-# Add files.
-ADD root/.bashrc /root/.bashrc
-ADD root/.gitconfig /root/.gitconfig
-ADD root/.scripts /root/.scripts
+# install generic build dependencies
+# also add multiverse (non-free) apps
+RUN sed -i 's/# \(.*multiverse$\)/\1/g' /etc/apt/sources.list && \
+    apt-get update && \
+    apt-get -y upgrade && \
+    # TODO git shouldn't be installed here,
```
Comment (bolshoy) [inline build/Dockerfile]: @oryband  why not? too much space? I think I saw other dockers assuming they have git

**Comment by bolshoy** [comment]
Why did you remove the scripts?

File: build/Dockerfile (hunk 1 above)
Comment (oryband) [inline build/Dockerfile]: maybe you're right
i thought having a `build` base image and `build-git` image, but i guess all our code for building stuff will come from git so maybe its redundant

**Pull request merged**